src/components/DateRangePicker.jsx

```diff
@@ -250,7 +250,8 @@ class DateRangePicker extends React.PureComponent {
     //
     // We handle both situations here by using the ` || ` operator to fallback
     // to *event.target** when **relatedTarget** is not provided.
```

Comment:
can we add a comment here explaining the document.body stuff for IE 11?

Reply:
@ljharb done